dashboard/config/foorm/library/surveys/pd/facilitator_panel.0.json

```diff
@@ -9,6 +9,11 @@
      "name": "facilitators",
      "title": "Facilitators",
      "templateElements": [
+      {
```

Comment:
should add configuration 
```
     "allowAddPanel": false,
     "allowRemovePanel": false
```
to the panel so teachers don't see a confusing add/remove option for facilitator questions

Reply:
lol.  I thought that was a feature you were tinkering with in the editor.  added.